src/assessments/keyboard-interaction/test-steps/character-key-shortcuts.tsx

```diff
@@ -18,7 +19,7 @@ const howToTest: JSX.Element = (
                 numbers, symbols, or punctuation. (Alt, Shift, and Ctrl are non-printable keys.)
             </li>
             <li>
-                Verify that individual keystrokes do not require specific timings, such as requiring users to:
+                Verify that at least one of the following is true:
```

Comment:
This reads a little awkwardly in combination with the list following it; "Remap."  or "Turn off." aren't really things that can be verified as "true or false". Maybe just removing the `Markup.Term`s in the list items would be more clear?

Reply:
Good question! I think @ferBonnin is probably the best person to answer to that.

Reply:
I see what you mean. I am fine with removing the markup.term